source/application/AVIDumping/AVIDumper.cpp

```diff
@@ -57,680 +57,754 @@ extern Movie movie;
 
 AudioConverterStream* audioConverterStream = NULL;
 
-HRESULT SafeAVIStreamWrite(PAVISTREAM pavi, LONG lStart, LONG lSamples, LPVOID lpBuffer, LONG cbBuffer, DWORD dwFlags, LONG FAR *plSampWritten, LONG FAR *plBytesWritten)
+HRESULT SafeAVIStreamWrite(PAVISTREAM pavi,
+                           LONG lStart,
+                           LONG lSamples,
+                           LPVOID lpBuffer,
+                           LONG cbBuffer,
+                           DWORD dwFlags,
+                           LONG FAR* plSampWritten,
+                           LONG FAR* plBytesWritten)
 {
-	HRESULT hr;
-
-	try
-	{
-		hr = AVIStreamWrite(pavi, lStart, lSamples, lpBuffer, cbBuffer, dwFlags, plSampWritten, plBytesWritten);
-	} 
-	catch (const Utils::Exceptions::WindowsException&)
-	{
-		hr = E_POINTER;
-	}
-	return hr;
+    HRESULT hr;
+
+    try
+    {
+        hr = AVIStreamWrite(
+            pavi, lStart, lSamples, lpBuffer, cbBuffer, dwFlags, plSampWritten, plBytesWritten);
+    }
+    catch (const Utils::Exceptions::WindowsException&)
+    {
+        hr = E_POINTER;
+    }
+    return hr;
 }
 
-//0x1FE -> -1
+// 0x1FE -> -1
 // 0xFF -> 0
 // 0x7F -> 1
 // 0x3F -> 2
 static int leftShiftFromMask(unsigned int mask)
 {
-	int shift = 0;
-
-	if (mask)
-	{
-		if (mask < 0xFF)
-		{
-			while ((mask << shift) < 128)
-			{
-				shift++;
-			}
-		}
-		else
-		{
-			while ((mask >> -shift) > 255)
-			{
-				shift--;
-			}
-		}
-	}
-
-	return shift;
+    int shift = 0;
+
+    if (mask)
+    {
+        if (mask < 0xFF)
+        {
+            while ((mask << shift) < 128)
+            {
+                shift++;
+            }
+        }
+        else
+        {
+            while ((mask >> -shift) > 255)
+            {
+                shift--;
+            }
+        }
+    }
+
+    return shift;
 }
 
 template<typename T>
 static void ReserveBuffer(T*& buffer, int& bufferAllocated, int size)
 {
-	if (!buffer || bufferAllocated < size)
-	{
-		bufferAllocated = size;
-		buffer = static_cast<T*>(realloc(buffer, size));
-	}
+    if (!buffer || bufferAllocated < size)
+    {
+        bufferAllocated = size;
+        buffer = static_cast<T*>(realloc(buffer, size));
+    }
 }
 
-// warning: gamma ramp is (unlike everything else) stored in a way that doesn't properly reset when loading a savestate.
+// warning: gamma ramp is (unlike everything else) stored in a way that doesn't properly reset when
+// loading a savestate.
 // this is probably fine since it's currently only used for AVI capture.
 DDGAMMARAMP g_gammaRamp;
 void SetGammaRamp(void* gammaRampPointer, HANDLE process)
 {
-	if (gammaRampPointer)
-	{
-		SIZE_T bytesRead = 0;
-
-		if (ReadProcessMemory(process, gammaRampPointer, &g_gammaRamp, sizeof(DDGAMMARAMP), &bytesRead))
-		{
-			g_gammaRampEnabled = true;
-		}
-	}
-	else
-	{
-		g_gammaRampEnabled = false;
-	}
+    if (gammaRampPointer)
+    {
+        SIZE_T bytesRead = 0;
+
+        if (ReadProcessMemory(
```

Comment:
Can we make it prioritize parameter breaking before this type of breaking?

Reply:
Should be done.